src/app/zap-templates/templates/app/callback.zapt

```diff
@@ -169,11 +171,17 @@ void emberAf{{asUpperCamelCase label}}ClusterClientTickCallback(chip::EndpointId
 
 {{#zcl_clusters}}
 {{#zcl_commands}}
+{{#if (isStrEqual source 'server')}}
 /**
-* @brief {{parent.name}} Cluster {{name}} Command callback
+* @brief {{parent.name}} Cluster {{name}} Command callback (client side)
 */
-bool emberAf{{asUpperCamelCase parent.label}}Cluster{{asUpperCamelCase name}}Callback(chip::app::Command * commandObj{{#zcl_command_arguments}}, {{asUnderlyingZclType type}} {{asSymbol label}}{{/zcl_command_arguments}});
-
+bool emberAf{{asUpperCamelCase parent.label}}Cluster{{asUpperCamelCase name}}Callback(chip::app::CommandSender * commandObj{{#zcl_command_arguments}}, {{asUnderlyingZclType type}} {{asSymbol label}}{{/zcl_command_arguments}});
+{{else if (isStrEqual source 'client')}}
+/**
+* @brief {{parent.name}} Cluster {{name}} Command callback (server side)
+*/
+bool emberAf{{asUpperCamelCase parent.label}}Cluster{{asUpperCamelCase name}}Callback(chip::app::CommandHandler * commandObj{{#zcl_command_arguments}}, {{asUnderlyingZclType type}} {{asSymbol label}}{{/zcl_command_arguments}});
+{{/if}}
```

Comment:
It can not be done with `chip_server_cluster_commands` because `chip_server_cluster_commands` will generates code based on the app configuration, while this file expects to generate declarations for every clusters.

But it can be rewritten such as:
```
/**
 * @brief {{parent.name}} Cluster {{name}} Command callback ({{source}} side)
 */
 bool emberAf{{asUpperCamelCase parent.label}}Cluster{{asUpperCamelCase name}}Callback(chip::app::Command{{#if (isServer source)}}Sender{{else}}Handler{{/if}} * commandObj{{#zcl_command_arguments}}, {{asUnderlyingZclType type}} {{asSymbol label}}{{/zcl_command_arguments}});
```